docs/config/front-matter.md

```diff
@@ -18,6 +35,11 @@ tags:
 ```
 
 ## date
+
+- Type: `YYYY-MM-DD`
```

Comment:
Will timezones be accepted as well? Best to document the standard if we're using ISO-8601 or something else

Reply:
VuePress' using `gray-matter` which depends `js-yaml` which follows standard yaml types, so all available formats are [here](https://yaml.org/type/timestamp.html). I've updated the docs.